src/ast/passes/codegen_llvm.h

```diff
@@ -99,6 +99,7 @@ class CodegenLLVM : public Visitor {
   libbpf::bpf_map_type get_map_type(const SizedType &val_type,
                                     const MapKey &key);
   void generate_maps(const RequiredResources &resources);
+  bool needs_percpu_map(const SizedType &val_type);
```

Comment:
It looks like this would be better placed in `types.h` as a method of `SizedType` (and probably will make sense to name it `needsPercpuMap` to comply with the rest of the class).